src/oid.rs

```diff
@@ -114,6 +114,15 @@ impl<'a> Oid<'a> {
         }
     }
 
+    /// Get the encoded oid without the header. Const version.
+    /// Only works when memory is not allocated on the heap, otherwise it returns an empty array.
+    pub const fn bytes_from_borrowed(&self) -> &[u8] {
+        match &self.asn1 {
+            Cow::Borrowed(b) => *b,
+            _ => &[],
```

Comment:
I don't know what the best behavior is:
- returning an empty slice hides the error, which may be bad
- this is a `const fn`, so we can't panic (an ugly way to do that would be to do something like `{"error message"[256]; &[]}` so at least we get an error message